### PR TITLE
Fixes #7523 - Removed autocomplete search from id fields

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: app/models/compute_resource.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ class ComputeResource < ActiveRecord::Base
   validates :url, :presence => true
   scoped_search :on => :name, :complete_value => :true
   scoped_search :on => :type, :complete_value => :true
-  scoped_search :on => :id, :complete_value => :true
+  scoped_search :on => :id, :complete_enabled => false
```
Comment (ohadlevy) [inline app/models/compute_resource.rb]: only_explicit too?

Comment (tbrisker) [inline app/models/compute_resource.rb]: @ohadlevy right, missed that one.

**Comment by ohadlevy** [comment]
missing the image model search, will add on merge.

**Comment by ohadlevy** [comment]
merged as 1b784c5 thanks @tbrisker

**Pull request closed without merging**